website/public/js/controllers/groupsCtrl.js

```diff
@@ -173,6 +173,7 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
           member.petCount = Shared.countPets($rootScope.countExists(member.items.pets), member.items.pets);
           member.mountCount = Shared.countMounts($rootScope.countExists(member.items.mounts), member.items.mounts);
         $scope.profile = member;
+        $scope.e = member.items.gear.equipped;
```

Comment:
I dunno; it would work, but it would make some of the lines pretty unwieldy...
```
{{#{g}[#{e}.weapon].#{k} + #{g}[#{e}.armor].#{k} + #{g}[#{e}.head].#{k} + #{g}[#{e}.shield].#{k} || 0}}
```
I'd prefer to put the variables on scope for code readability. That being said, assuming we're talking about removing the `ng-init` altogether, putting the variables on scope would require a refactor to ensure that `stats.jade` works on the profile page as well as the member modal. So in a way, jade variables might be easier, even if it makes the code uglier. What do you think?